pages/linux/sa.md

```diff
@@ -0,0 +1,16 @@
+# sa
+
+> Summarizes accounting information. Part of the acct package.
```

Comment:
Is really important to say that it is a part of the acct package?

Reply:
Hrm, I'm not sure. If we change this then we'll have to go back over the others that are also part of `acct` that have been contributed recently.

Reply:
In my opinion, the package does not really matter. 

Not a problem for it to be there, though. 
I think we can merge this as it is.